handler/instrumented_enqueue_object_test.go

```diff
@@ -154,15 +154,8 @@ var _ = Describe("InstrumentedEnqueueRequestForObject", func() {
 			instance.Update(evt, q)
 
 			// verify workqueue
-			Expect(q.Len()).To(Equal(2))
+			Expect(q.Len()).To(Equal(1))
```

Comment:
this is because of the deduplication of events before being sent into work queue.
ref: https://github.com/kubernetes-sigs/controller-runtime/pull/1390